experiments/models.py

```diff
@@ -125,3 +284,34 @@ class Meta:
         unique_together = [
             ('experiment', 'date', 'variation'),
         ]
+
+        verbose_name = _('Experiment History')
+        verbose_name_plural = _('Experiment Histories')
+
+if settings.DEBUG:
+    class SimplePage(Page):
```

Comment:
This does not belong here, but https://github.com/torchbox/wagtail-experiments/blob/master/tests/models.py

We should not add test models just because someone runs their project with DEBUG=True (for example, locally)

Reply:
That was an oversight. I'll be sure to fix it. 

Thanks for the thorough review, zerolab!